spec/factories/zines.rb

```diff
@@ -1,8 +1,8 @@
 FactoryGirl.define do
 
   factory :zine do
-    title Faker::Lorem.words.join(' ').capitalize!
-    subtitle Faker::Lorem.words.join(' ').capitalize!
+    title { Faker::Lorem.words.join(' ').capitalize! }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.